misc/fuzzer_config.txt

```diff
@@ -0,0 +1,55 @@
+use_parent_methods: true
+add_to_tree: true
+delay_removing_added_nodes_to_next_frame: true
+test_one_class_multiple_times: true
+
+custom_classes:
+FastNoiseLite
+FastNoiseLiteGradient
+Voxel
+VoxelBlockSerializer
+VoxelBoxMover
+VoxelBuffer
+VoxelColorPalette
+VoxelGenerator
+VoxelGeneratorFlat
+VoxelGeneratorGraph
+VoxelGeneratorHeightmap
+VoxelGeneratorImage
+VoxelGeneratorNoise
+VoxelGeneratorNoise2D
+VoxelGeneratorScript
+VoxelGeneratorWaves
+VoxelInstanceComponent
+VoxelInstanceGenerator
+VoxelInstanceLibrary
+VoxelInstanceLibraryItem
+VoxelInstanceLibraryItemBase
+VoxelInstanceLibrarySceneItem
+VoxelInstancer
+VoxelLibrary
+VoxelLodTerrain
+VoxelMesher
+VoxelMesherBlocky
+VoxelMesherCubes
+VoxelMesherDMC
+VoxelMesherTransvoxel
+VoxelNode
+VoxelRaycastResult
+VoxelServer
+VoxelStream
+VoxelStreamBlockFiles
+VoxelStreamRegionFiles
+VoxelStreamSQLite
+VoxelStreamScript
+VoxelTerrain
+VoxelTool
+VoxelToolBuffer
+VoxelToolLodTerrain
+VoxelToolTerrain
+#VoxelViewer # Cause freeze when using with VoxelTerrain, due iterating each cell
```

Comment:
What do you mean "iterating each cell"?
Overall, there are some setups that a CI cannot possibly run with a fuzzer. One example is when a fuzzer happens to set LOD count = 1 and a view distance of 1000. This is legit assuming the computer is beefy enough (likely not), but it can bring the machine to its knees.

Reply:
Like I wrote in  #330, running `VoxelTerrain` and `VoxelViewer` freeze when both running(I couldn't find steps to reproduce) due calculating somethings in `process_viewers` function.
Setting `test_one_class_multiple_times` probably should fix this and `VoxelViewer` probably could be uncommented, but I'm not sure in 100%.